ansible/group_vars/nim-waku-connect.yml

```diff
@@ -46,5 +46,6 @@ open_ports_list:
   - { port: '{{ nim_waku_p2p_tcp_port }}' }
   - { port: '{{ nim_waku_p2p_udp_port }}', protocol: 'udp' }
   - { port: '{{ nim_waku_websock_port }}' }
+  - { port: '{{ nim_waku_websocket_cont_port }}' }
```

Comment:
There is no such variable as `nim_waku_websocket_cont_port` in the role:
https://github.com/status-im/infra-role-nim-waku/blob/master/defaults/main.yml